ibc-clients/ics07-tendermint/types/src/error.rs

```diff
@@ -60,8 +60,7 @@ impl std::error::Error for TendermintClientError {
     }
 }
 
-// TODO(seanchen1991): Should this impl be deprecated in favor of a
-// From<ClientError> for TendermintClientError impl?
+// TODO(seanchen1991): Figure out how to remove this conversion
```

Comment:
Let's remove this TODO (there is another in the `ClientError` as well) and track it via a GitHub issue.